pom.xml

```diff
@@ -339,6 +339,25 @@
         </plugins>
       </reporting>
     </profile>
+    <profile>
+      <id>surefire-configuration</id>
+      <activation>
+        <property>
+          <name>${test.arguments}</name>
```

Comment:
Just noticed this is evaluating the property which is wrong. The activation tag should look like

`<name>test.arguments</name>`

Reply:
Good spot!